crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -22,7 +22,7 @@
           "name": "Write API"
         }
       ],
-      "description": "Runs the transaction in dev-inspect mode. Which allows for nearly any transaction (or Move call) with any arguments. Detailed results are provided, including both the transaction effects and any return values.",
+      "description": "Simulates running the transaction, even one that would be normally impossible, against the current state of the network. Nearly any transaction (or Move call) with any arguments can run in dev-inspect; transaction inputs are not checked, entry function verification is skipped, values can be left unused, and returns are not validated. Detailed results are provided, including both the transaction effects and any return values. Dev-inspect is provided to aid in understanding the effects of any possible action against the current network state. See dryRunTransactionBlock for a more faithful simulation of what would happen if the transaction were actually run.",
```

Comment:
These docs are auto-generated, e.g. this one comes from here:

https://github.com/MystenLabs/sui/blob/37ddeb6a2ae1b0ee18260e3c582ac718378ce74b/crates/sui-json-rpc/src/api/write.rs#L43

Could you update the docs there and then run:

```
cargo -q run --example generate-json-rpc-spec --record
```

To port them over here?

Otherwise your changes will get lost with the next RPC change, unfortunately!

My tactic is to search for the existing doc snippets in rust files in the repo to find where they originate from.

Reply:
Thanks! I'll make that change.

Reply:
(Just for posterity, the command is actually `cargo -q run --example generate-json-rpc-spec -- record`.)